### PR TITLE
Add checklist of pre-tasks for Final Project

### PR DESCRIPTION
**Pull request by kfklein15**
We have identified that a checklist of the Final Project tasks that must be done way before this module would be handy for PROMs and Volunteers. So I have added this here. PS: I will review the rest of the page too, so will have a second PR soon.

Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): Preparation

## Description

 This PR includes a checklist of all tasks that should be done for Final Project starting months before the actual module, to ensure we are all following a similar process

## Who needs to know about this? 

 @ElizabethlawalCYF @MarianaVazquezCYF @martaesn @AliciaE08 @ShaunCYFSA @caromelendezl 

## Rendered Pages

-----
[View rendered docs/finalproject/prep.md](https://github.com/CodeYourFuture/syllabus/blob/kfklein15-patch-1/docs/finalproject/prep.md)

### CODE REVIEW

File: docs/finalproject/prep.md [hunk 1]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
...
+- [ ] Have a meeting with all volunteers to go through the details of the final project (presentation to be created with expectation, escalation process, type of support, overview of the trainees per team, etc)
+- [ ] Introduce all teams and people in it and create all Slack groups
+- [ ] A day before the first Saturday of Final Projects, send the project briefing to the team
+
 ## Project Submissions
 
-Some trainees may have some ideas about projects that they would like to complete for their Final Project. Before the Final Project starts, ideas should be gathered and fully specified before the start of the first week.
+As mentioned above, before the Final Project starts ideas should be gathered and fully specified before the start of the first week. The form used to receive these ideas is [this](https://forms.gle/Gn85Kdna6QB2Qfcp9). 
 
 A good project should
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:37]: Seems to cut off here, @kfklein15 ?

Comment (SallyMcGrath) [reply docs/finalproject/prep.md:37]: Oh doh ignore me. Teach me to code review on flaky internet!

File: docs/finalproject/prep.md [hunk 2]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
 sidebar_label: Preparation
 ---
 
+## Checklist to organise Final Projects
+Getting the Final Projects ready is not a task that can be done a week before the module. A lot of decisions have to be made beforehand, so this high-level list of the steps Volunteers and PROM (Programme Managers) should do in order to get everything ready for Final Projects. It also has the main links of important documents. 
+
+**When you are on the React Module**
+- [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
+- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:12]: I think this happens at the end of JS3 or the trainees won't have 12 weeks to specialise in their new role

File: docs/finalproject/prep.md [hunk 3]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
 sidebar_label: Preparation
 ---
 
+## Checklist to organise Final Projects
+Getting the Final Projects ready is not a task that can be done a week before the module. A lot of decisions have to be made beforehand, so this high-level list of the steps Volunteers and PROM (Programme Managers) should do in order to get everything ready for Final Projects. It also has the main links of important documents. 
+
+**When you are on the React Module**
+- [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:11]: I recommend sharing the entry and exit criteria in HTML, JS3, React, Node, DB

Trainees have a huge amount of stuff to pay attention to, so I'd like us to help them by returning their attention to this a few times.

File: docs/finalproject/prep.md [hunk 4]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
...
+- [ ] Review the project ideas with Tech Leads and Product Managers defining technical feasibility and scope size. You might have to have volunteers clarify some information with the charities
+- [ ] Identify which of the Ideas will be used as a final project. 
+	Send feedback to all charities if they have been chosen or not. 
+	For the chosen ones, clarify the [role](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-owner) and who will be the Product Owner from their Charity. 
+- [ ] Onboard the volunteers with the standard process
+- [ ] Use [this template](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing) to identify the projects that will be used and who will be on which team
+- [ ] Let volunteers choose which project they would like to be on
+
+**When you are on SQL Module**
+- [ ] Trainees must have finalised the Full Stack assessment
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:26]: Hm, maybe something closer to "Trainees must have completed up to Level 300 by week 3 of the module before Final Projects. " ?

Here's some more info about this that might help us rephrase this to get this a little clearer.  They have three weeks to work on the Full Stack Project Assessment.

https://github.com/CodeYourFuture/Full-Stack-Project-Assessment 

Each week has a mandatory minimum level they have to achieve and someone needs to check this has happened. There is a week 4 in the DB module that volunteers have "spare" to pick up any dropped threads on this but trainees have to be at level 300 by week 3.

There are also various optional stretch levels, but doing random parts of the stretch tasks and not completing the core task does not comprise a pass, alas. Levels 100, 200, 250, and 300 must be complete to enter Final Projects in a full stack dev role.

File: docs/finalproject/prep.md [hunk 5]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
...
+- [ ] Identify which of the Ideas will be used as a final project. 
+	Send feedback to all charities if they have been chosen or not. 
+	For the chosen ones, clarify the [role](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-owner) and who will be the Product Owner from their Charity. 
+- [ ] Onboard the volunteers with the standard process
+- [ ] Use [this template](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing) to identify the projects that will be used and who will be on which team
+- [ ] Let volunteers choose which project they would like to be on
+
+**When you are on SQL Module**
+- [ ] Trainees must have finalised the Full Stack assessment 
+- [ ] Must review the trainee’s performance and identify the ones going to Final Projects
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:27]: trainees'

(plural possessive)

File: docs/finalproject/prep.md [hunk 6]
```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
...
+- [ ] Identify trainees that are close in performance and allocate them to the same project. We encourage trainees to work with different people, so try to mix the teams up. Don’t forget to update the spreadsheet.
+- [ ] Have a meeting with all volunteers to go through the details of the final project (presentation to be created with expectation, escalation process, type of support, overview of the trainees per team, etc)
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md:29]: Suggest _all hands,_ trainees and volunteers together (and record for the 50% of people that can't make it)

**Review by SallyMcGrath — APPROVED** [review]
Thanks for this, @kfklein15 

I've left a couple of comments for you to consider, but also approved as my notes for improvement are not blocking

File: docs/finalproject/prep.md (hunk 3 above)
Comment (kfklein15) [reply docs/finalproject/prep.md:11]: Agree. Maybe we should add this to the HTML/JS3/React/Node/DB syllabus, so volunteers remember to do it?
